src/lib/accessibility/types.ts

```diff
@@ -102,6 +102,7 @@ export interface AriaWidgetAttributes {
   'aria-valuemin'?: string
   'aria-valuenow'?: string
   'aria-valuetext'?: string
+  'aria-modal'?: boolean
```

Comment:
lets fit alphabetical order with this one

Reply:
right, thank you!